dcm4che-assembly/src/etc/findscu/patient.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<NativeDicomModel xml:space="preserve">
+    <DicomAttribute keyword="SpecificCharacterSet" tag="00080005" vr="CS">
+        <Value number="1">ISO_IR 100</Value>
+    </DicomAttribute>
+    <DicomAttribute keyword="QueryRetrieveLevel" tag="00080052" vr="CS">
+        <Value number="1">PATIENT</Value>
+    </DicomAttribute>
+    <DicomAttribute keyword="PatientName" tag="00100010" vr="PN">
+        <PersonName number="1">
+            <Alphabetic>
+                <FamilyName>Ariel</FamilyName>
```

Comment:
(Maybe it is not the best idea to use the name of famous politicians as sample data.)